setup/install/halyard.md

```diff
@@ -70,7 +70,12 @@ your container:
 
    You can interact with Halyard from here.
 
-1. Run `. ~/.bashrc` to enable command completion.
+1. Run the following commands to enable command completion:
+
+   ```bash
+   hal --print-bash-completion > ~/hal_bash_completion
+   source ~/hal_bash_completion
```

Comment:
I think you can remove this file as the last command `rm ~/hal_bash_completion`. There is no need to keep it & it'll just clutter the users home directory.

Reply:
Second that.

Looks like you can run ```source <(hal --print-bash-completion)``` and avoid having to create the file altogether.